inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/repositories/LocalEventRepository.kt

```diff
@@ -119,6 +126,17 @@ internal interface LocalEventRepository : EventRepository {
             }
         }
 
+        /**
+         * {@inheritDoc}.
+         * */
+        override fun clearNonPersistentEvents(timeMillis: Long) {
```

Comment:
Is there a way to combine this with the first `clearNonPersistentEvents()`?

Reply:
Because it takes a newer time compared to created event time to satisfy the default condition (deleting all events) we will not be able to use a default fixed time.
One solution will be adding check conditions to calculate the newer date and put it as default, but it will make the code more complicated than just creation a new method.